corehq/apps/data_analytics/models.py

```diff
@@ -15,7 +15,8 @@ class MALTRow(models.Model):
     user_id = models.TextField()
     username = models.TextField()
     email = models.EmailField()
-    is_web_user = models.BooleanField(default=False)
+    user_type = models.TextField()
```

Comment:
Oh nice - that sounds like a good idea.  I guess the Django ORM could do with an EnumField, huh?